examples/swaggercodegen/api/resources/swagger.yaml

```diff
@@ -171,6 +171,65 @@ paths:
       security:
         - apikey_auth: []
 
+  ## CDN sub-resource
+
+  /v1/cdns/{parent_id}/v1/firewalls:
+    post:
+      x-terraform-resource-name: "cdn_v1_firewalls"
+      x-terraform-resource-host: localhost:8443 # If this extension is specified, it will override the global host and API calls will be made against this host instead
+      tags:
+      - "cdn"
+      summary: "Create cdn firewall"
+      operationId: "ContentDeliveryNetworkFirewallCreateV1"
+      parameters:
+      - name: "parent_id"
+        in: "path"
+        description: "The cdn id that contains the firewall to be fetched."
+        required: true
+        type: "string"
+      - in: "body"
+        name: "body"
+        description: "Created CDN firewall"
+        required: true
+        schema:
+          $ref: "#/definitions/ContentDeliveryNetworkFirewallV1"
+      responses:
+        201:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/ContentDeliveryNetworkFirewallV1"
+        default:
+          description: "generic error response"
+          schema:
+            $ref: "#/definitions/Error"
+  /v1/cdns/{parent_id}/firewalls/{id}:
+    get:
+      tags:
+      - "cdn"
```

Comment:
I guess it could, it's a subresource of cdn so kind of makes sense also to be in the same tag...just an example feel free to update if you feel strong about it

Reply:
No strong feeling about it.  Just a thought.